frontend/public/components/resource-pages.ts

```diff
@@ -91,6 +93,7 @@ export const resourceDetailPages = ImmutableMap<GroupVersionKind | string, () =>
 
 export const resourceListPages = ImmutableMap<GroupVersionKind | string, () => Promise<React.ComponentType<any>>>()
   .set(referenceForModel(ClusterModel), () => import('./clusters' /* webpackChunkName: "clusters" */).then(m => m.ClustersPage))
+  .set(referenceForModel(ClusterServiceClassModel), () => import('./cluster-service-classes' /* webpackChunkName: "cluster-service-classes" */).then(m => m.ServiceClassesPage))
```

Comment:
I'd call the module `cluster-service-class` and use `/* webpackChunkName: "cluster-service-class" */` since almost all of the other resources are singular. (`clusters` just above seems to be the exception.)